### DISCUSSION TITLE
Plotting ISCE outputs (ISCE to PNG)

### DISCUSSION THREAD

**Discussion by camurad12** [isce-framework/isce2#484] [Q&A]
I'm processing Sentinel 1 data with ISCE. I have already run the software, but in order to assess the results I obtained, I would like to visualize the files on the /merged folder that is created. For example, to translate the file filt_topophase.flat.geo to filt_topophase.flat.geo.png, as shown in the image below.

![inter](https://user-images.githubusercontent.com/96483992/166982721-c610b5ee-353b-449d-b520-06d87c2b9c05.png)

I'm just getting a grasp on python so I'm a bit lost on how to achieve this. 
I have also been able to read the outputs in  MATLAB but haven't been able to properly plot them.
I will really appreciate if you can guide me through it.

**Comment by riyasmjgeo** [comment]
Can use GDAL for the conversion. Example code is `gdal_translate -of PNG topophase.flat output.png`
But be aware of the data type.
